e2e/e2e_test.go

```diff
@@ -61,6 +62,14 @@ var _ = Describe("Multus dynamic networks controller", func() {
 			})
 		}
 
+		listPodsOnNamespace := func() []corev1.Pod {
+			pods, err := clients.ListPods(namespace, appLabel(podName))
```

Comment:
error can be etcd timeout basically, in this case you need to rerun, 
returning nil may be too early

Reply:
I'm thinking of actually making this less generic, and ask what I really want to know:
```golang
		isTestNamespaceEmpty := func() bool {
			pods, err := clients.ListPods(namespace, appLabel(podName))
			if err != nil {
				return false
			}
			return len(pods.Items) == 0
		}
```